python/ui.py

```diff
@@ -157,21 +157,22 @@ def render(changed):
             if fn:
                 with open(fn, "w") as f:
                     f.write(tensor.code)
-            _ = benchmark(tensor, 10)  # warmup
+            # _ = benchmark(tensor, 10)  # warmup
```

Comment:
feel free to just delete these lines entirely

Reply:
I wanted to discuss how we benchmark things. I am not sure my approach is better than the previous one.